config/workflows/templates/15/job.xml

```diff
@@ -0,0 +1,331 @@
+# Prepare container working directory 
+containerWorkingDirectory = '-w '+workspaceContainer+' ' 
+# Save pre execution command into magic variable 'preJavaHomeCmd', which is picked up by the node 
+preJavaHomeCmd = dockerRunCommand + dockerParameters + proActiveHomeVolume + workspaceVolume + containerWorkingDirectory + containerName
+]]>
+            </code>
+          </script>
+        </envScript>
+      </forkEnvironment>
+      <pre>
+        <script>
+          <code language="bash">
+            <![CDATA[
+
+]]>
+          </code>
+        </script>
+      </pre>
+      <scriptExecutable>
+        <script>
+          <code language="Language R">
+            <![CDATA[
+#Change to your own or the limits will be shared
+consumer_key_api <- "K5sM7uI8QPwK1iiTsvWayIl9q"
+consumer_secret_api <- "4Lc9V9bjzNTI2B6Eq7wvf3TZJQMnv3sFqiDCBCgqWQL1kjFmM6"
+token <- "804452393895088129-JmyRNVoDLaDBOyjlUrh4my3Yl8M2Q9U"
+token_secret <- "yedHoQyQEardwoQNbFraWIz1n552BgfADDk5XOSwcc7PT"
+#15 calls every 15 minutes. 
+
+
+
+Locations <- data.frame( lats=unlist(results[[1]][1]), lons=unlist(results[[1]][2]), cities=unlist(results[[1]][3]))
+
+
+Sys.setlocale("LC_NUMERIC", "C") 
+library(twitteR)
+library(stringr)
+library(tm)
+library(wordcloud)
+library(textreg)
+library(mailR)
+
+
+tryCatch({
+options(httr_oauth_cache=F)
+setup_twitter_oauth(consumer_key_api, consumer_secret_api, token , token_secret)
+} ,error=function(cond) {
+print("The credentials seem to be wrong or you made too many calls. In the latter case, wait 15 minutes and try again")
+stop("Could not connect")
+return(NA)
+})
+
+Tweets <- searchTwitter(toString(variables[["KeyWord"]]) , n=variables[["Number_of_tweets"]], lang='en', geocode=paste(Locations$lats[variables[["PA_TASK_REPLICATION"]]+1],Locations$lons[variables[["PA_TASK_REPLICATION"]]+1],paste0(300,"km"),sep=",") )
+
+
+Tweetstext <-sapply(Tweets, function(x) x$getText())
+Tweetstext <-unlist(Tweetstext)
+
+
+corpus <- Corpus(VectorSource(Tweetstext))
+corpus <- tm_map(corpus, str_replace_all,"[^[:alnum:]]", " ")
+corpus <- tm_map(corpus, removePunctuation)
+corpus <- tm_map(corpus, removeWords, stopwords("english"))
+corpus <- tm_map(corpus, removeWords, c("http", "https", "amp", variables[["KeyWord"]]))
+corpus <- tm_map(corpus, PlainTextDocument)
+
+
+
+#plot and save
+col <- brewer.pal(6,"Dark2")
+png(paste(Locations$cities[variables[["PA_TASK_REPLICATION"]]+1], "_", variables[["KeyWord"]], ".png", sep = ""))
+plot.new()
+wordcloud(corpus, scale=c(3,.2),rot.per = 0.25,  random.color=T, max.word=700, random.order=F, colors=col)
+dev.off()
+
+
+
+corpus <- convert.tm.to.character(corpus)
+result <- corpus
+]]>
+          </code>
+        </script>
+      </scriptExecutable>
+      <controlFlow block="start"></controlFlow>
+      <outputFiles>
+        <files  includes="*.png" accessMode="transferToUserSpace"/>
+      </outputFiles>
+    </task>
+    <task name="Split">
+      <forkEnvironment javaHome="/usr" >
+        <envScript>
```

Comment:
Is it necessary to use this _/usr_ prefix? is it not possible to rely on `JAVA_HOME` env variable?